docs/pages/access-controls/guides/role-templates.mdx

```diff
@@ -325,3 +325,47 @@ Function | Description
 --- | ---
 `email.local(variable)` | Extracts the local part of an email field, like `Alice <alice@example.com>` or `bob@example.com`.
 `regexp.replace(variable, expression, replacement)` | Finds all matches of `expression` and replaces them with `replacement`. This supports expansion, e.g. `regexp.replace(external.email, "^(.*)@example.com$", "$1")`. Values which do not match the expression will be filtered out.
+
+## Pattern matching in Access & Review Requests
```

Comment:
Let's shorten this to `Templating in Access Requests`

Reply:
Done